Tests/test_lib_image.py

```diff
@@ -1,7 +1,37 @@
+import secrets
+
 import pytest
 
 from PIL import Image
 
+mode_names_not_bgr = (
+    "1",
+    "L",
+    "LA",
+    "La",
+    "P",
+    "PA",
+    "F",
+    "I",
+    "I;16",
+    "I;16L",
+    "I;16B",
+    "I;16N",
+    "RGB",
+    "RGBA",
+    "RGBa",
+    "RGBX",
+    # Image.frombytes() doesn't work with BGR modes:
+    # unknown raw mode for given image mode
+    # "BGR;15",
+    # "BGR;16",
+    # "BGR;24",
```

Comment:
I fixed these errors in #7303, but have since discovered that is not sufficient to get this PR working for those modes. The problem is that those modes are using their lines efficiently, rather than spacing out their data into 4 bytes per pixel.
https://github.com/python-pillow/Pillow/blob/96d683dafa3bb57562d0ba8bf887e18a8b8d9125/src/libImaging/Storage.c#L149-L150